rotkehlchen/tests/exchanges/test_ftx.py

```diff
@@ -0,0 +1,567 @@
+        expected_warnings_num=0,
+        expected_errors_num=0,
+        expected_actions_num=3,
+    )
+
+    # Invalid time
+    broken_time = FILLS_RESPONSE.replace('2021-03-11T01:01:01.662266+00:00', 'true')
+    query_ftx_and_test(
+        ftx=mock_ftx,
+        fills_response=broken_time,
+        query_fn_name='query_online_trade_history',
+        expected_warnings_num=0,
+        expected_errors_num=1,
+        expected_actions_num=2,
+    )
+
+    # Unknown assets
+    broken_time = FILLS_RESPONSE.replace('USD', 'oldmoney')
+    query_ftx_and_test(
+        ftx=mock_ftx,
+        fills_response=broken_time,
+        query_fn_name='query_online_trade_history',
+        expected_warnings_num=1,
+        expected_errors_num=0,
+        expected_actions_num=2,
+    )
+
+    # Invalid asset format
+    broken_time = FILLS_RESPONSE.replace('USD', '123')
+    query_ftx_and_test(
+        ftx=mock_ftx,
+        fills_response=broken_time,
+        query_fn_name='query_online_trade_history',
+        expected_warnings_num=1,
+        expected_errors_num=0,
+        expected_actions_num=2,
+    )
+
+    # Invalid trade types
+    broken_time = FILLS_RESPONSE.replace('buy', 'investmentss')
+    query_ftx_and_test(
+        ftx=mock_ftx,
+        fills_response=broken_time,
+        query_fn_name='query_online_trade_history',
+        expected_warnings_num=0,
+        expected_errors_num=1,
+        expected_actions_num=2,
+    )
+
+    # Invalid amount
+    broken_time = FILLS_RESPONSE.replace('400', '"four hundred"')
+    query_ftx_and_test(
+        ftx=mock_ftx,
+        fills_response=broken_time,
+        query_fn_name='query_online_trade_history',
+        expected_warnings_num=0,
+        expected_errors_num=1,
+        expected_actions_num=2,
+    )
+
+    # Invalid fee amount
+    broken_time = FILLS_RESPONSE.replace('0.6', '"zero dot six"')
+    query_ftx_and_test(
+        ftx=mock_ftx,
+        fills_response=broken_time,
+        query_fn_name='query_online_trade_history',
+        expected_warnings_num=0,
+        expected_errors_num=1,
+        expected_actions_num=2,
+    )
+
+    # Unknown fee asset
+    broken_time = FILLS_RESPONSE.replace('BTC', 'AUGMENTED')
+    query_ftx_and_test(
+        ftx=mock_ftx,
+        fills_response=broken_time,
+        query_fn_name='query_online_trade_history',
+        expected_warnings_num=1,
+        expected_errors_num=0,
+        expected_actions_num=2,
+    )
+
+    # missing key
+    broken_time = FILLS_RESPONSE.replace('id', 'ides')
+    query_ftx_and_test(
+        ftx=mock_ftx,
+        fills_response=broken_time,
+        query_fn_name='query_online_trade_history',
+        expected_warnings_num=0,
+        expected_errors_num=3,
+        expected_actions_num=0,
+    )
+
+
+@pytest.mark.parametrize('mocked_current_prices', [{"USDC": FVal(1), "ETH": FVal(2000)}])
+def test_balances(mock_ftx):
+    """Test that the mocked balances are correctly mocked"""
```

Comment:
> Test that the mocked balances are correctly mocked

Lol :rofl: 

Testing a mock makes no sense ever. I will just assume this is a typo and you want to say that you want to test the FTX balances response.

Reply:
Yes typo :skull:

Reply:
It's still there.